quinn/examples/README.md

```diff
@@ -0,0 +1,83 @@
+## HTTPS File Serving Example
+
+The `https` example shows how to request a file on the `server` from the `client` over HTTPS. 
```

Comment:
We should clarify that this is HTTP/0.9, which is a toy protocol that's much simpler than real HTTP, especially HTTP/3.